game/src/ecs/components/PositionComponent.java

```diff
@@ -26,12 +26,26 @@ public PositionComponent(@DSLContextMember(name = "entity") Entity entity, Point
     }
 
     /**
+     * @param entity associated entity
+     * @param x x-position of the entity
+     * @param y y-position of the entity
+     */
+    public PositionComponent(@DSLContextMember(name = "entity") Entity entity, float x, float y) {
+        this(entity, new Point(x, y));
+    }
+
+    /**
+     * Set the position of this entity on a random floor tile in the level. if no level is loaded,
+     * set the position to (0,0)
+     *
      * @param entity associated entity
      */
     public PositionComponent(@DSLContextMember(name = "entity") Entity entity) {
         super(entity);
-        this.position =
-                Game.currentLevel.getRandomTile(LevelElement.FLOOR).getCoordinate().toPoint();
+        if (Game.currentLevel != null)
```

Comment:
Wäre nicht `(0,0)` in beiden Fällen besser? Wer zufällig will, kann sich ja den zufälligen Punkt holen und damit den Ctor füttern?

Reply:
Das Ding ist (0,0) liegt nicht unbedingt im spielbaren level, es könnte wand oder "Außenbereich" sein.
Und ich würde jetzt mal behaupten, dass ich eine Entität (wie ein Monster) eher zufällig im spielbaren level platzieren möchte als auf 0,0.
Ich würde also sagen, wer unbedingt 0,0 will, soll das halt angeben.

Reply:
Hmm. Dann macht Dein Vorschlag Sinn. 

Schreibst Du das bitte als Begründung dann noch mit dazu? (Kommentar)